src/dawn/Compiler/Options.inc

```diff
@@ -72,6 +72,10 @@ OPT(bool, UseParallelEP, false, "use-parallel-ep", "",
     "Make use of the parallel execution policy", "", false, true)
 OPT(bool, UseKCaches, true, "use-kcaches", "", 
     "Make use of the k-caches", "", false, true)
+OPT(bool, PassTmpToFunction, false, "pass-tmp-to-function", "",
+    "Activate pass to replace temporary precomputations by stencil function calls", "", false, true)
+OPT(bool, ReportPassTmpToFunction, false, "report-pass-tmp-to-function", "",
```

Comment:
yes, that could make sense. Although for debugging also might be useful to keep the level only to the pass you are interested. Similar approach is done in gcc. Although we can always have a flag that enables all reports

Reply:
so how would you handle it? If we still keep the individual ones in, we have a lot of options (which might decrease accessibility to the more important ones)

An enable-all flag seems like it could help

Reply:
I created an issue